src/abstractdataframe/abstractdataframe.jl

```diff
@@ -674,7 +674,7 @@ function _describe(df::AbstractDataFrame, stats::AbstractVector)
         splice!(stats, i, allowed_fields) # insert in the stats vector to get a good order
     elseif predefined_funs == [:detailed]
         predefined_funs = [:mean, :std, :min, :q25, :median, :q75,
-                           :max, :nunique, :nmissing, :eltype]
+                           :max, :sum, :nunique, :nmissing, :eltype]
```

Comment:
I think `:sum` should not be added to `:detailed` option as it would be a breaking change.